.github/workflows/release.yaml

```diff
@@ -41,6 +41,7 @@ jobs:
           tag: ${{ env.RELEASE_TAG }}
           name: PHPUnit ${{ env.RELEASE_TAG }}
           bodyFile: release-notes.md
+          commit: "8.5"
```

Comment:
Unless we can find a way to obtain this from the GitHub context (see [`push`](https://docs.github.com/en/webhooks/webhook-events-and-payloads#push) event), we need to hard-code this value here (and update in other branches).

Reply:
> we need to hard-code this value here (and update in other branches).

...or

`BRANCH_NAME: ${{ github.ref_name }} `

| Property name |Description |
|--------|--------|
| `github.ref_name` | The short ref name of the branch or tag that triggered the workflow run. This value matches the branch or tag name shown on GitHub. For example, `feature-branch-1`.| 

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context